### PR TITLE
Remove "simple" part 2: change to "basic" 

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/css/_colon_where/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Take the following HTML:
 
 In this somewhat-contrived example, we have two articles that each contain a section, an aside, and a footer. They differ by the classes used to mark the child elements.
 
-To make selecting the links inside them simpler, but still distinct, we _could_ use `:is()` or `:where()`, in the following manner:
+To group the selection of links, while keeping the `is-styling` and `where-styling` styles distinct, we _could_ use `:is()` or `:where()`, in the following manner:
```
Comment (wbamberg) [inline files/en-us/web/css/_colon_where/index.md:109]: Again could keep "simpler" here IMO.

File: files/en-us/web/css/_colon_where/index.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ html {
 }
 ```
 
-However, what if we later want to override the color of links in the footers using a simple selector?
+However, what if we later want to override the color of links in the footers using a compound selector made up of low-specificity type selectors?
```
Comment (wbamberg) [inline files/en-us/web/css/_colon_where/index.md:126]: Is "simple selector" a technical term here? i.e. does it actually mean something?

File: files/en-us/web/css/_doublecolon_before/index.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Using a `::before` pseudo-element to add content is discouraged, as it is not re
 
 ### Adding quotation marks
 
-One simple example of using `::before` pseudo-elements is to provide quotation marks. Here we use both `::before` and {{Cssxref("::after")}} to insert quotation characters.
+One basic example of using `::before` pseudo-elements is to provide quotation marks. Here we use both `::before` and {{Cssxref("::after")}} to insert quotation characters.
```
Comment (wbamberg) [inline files/en-us/web/css/_doublecolon_before/index.md:38]:
```suggestion
One example of using `::before` pseudo-elements is to provide quotation marks. Here we use both `::before` and {{Cssxref("::after")}} to insert quotation characters.
```

File: files/en-us/web/css/_doublecolon_before/index.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ We can style text or images in the {{cssxref("content")}} property almost any wa
 
 ### To-do list
 
-In this example we will create a simple to-do list using pseudo-elements. This method can often be used to add small touches to the UI and improve user experience.
+In this example we will create a short to-do list using pseudo-elements. This method can often be used to add small touches to the UI and improve user experience.
```
Comment (wbamberg) [inline files/en-us/web/css/_doublecolon_before/index.md:95]: Could also just omit it here.

File: files/en-us/web/css/_colon_where/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/_colon_where/index.md:109]: with just an `a`, in this case grouping doesn't reduce any characters, so I don't think this is simpler in this case.

File: files/en-us/web/css/_colon_where/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/css/_colon_where/index.md:126]: yeah, a simple selector is just one selector. `#myId` and `::selection` are both simple selectors, while `p a` is not.

File: files/en-us/web/css/_doublecolon_before/index.md (hunk 3 above)
Comment (estelle) [inline files/en-us/web/css/_doublecolon_before/index.md:38]:
```suggestion
One example of using `::before` pseudo-elements is to provide quotation marks. Here we use both `::before` and {{Cssxref("::after")}} to insert quotation characters.
```

File: files/en-us/web/css/_doublecolon_before/index.md (hunk 4 above)
Comment (estelle) [inline files/en-us/web/css/_doublecolon_before/index.md:95]:
```suggestion
In this example we will create a to-do list using pseudo-elements. This method can often be used to add small touches to the UI and improve user experience.
```

**Review by estelle — COMMENTED** [review]
Thank you for the review.

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**